scripts/benchmark_test.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+# This script runs a benchmark on a locally started etcd server
+
+set -euo pipefail
+
+source ./scripts/test_lib.sh
+
+COMMON_BENCHMARK_FLAGS="--report-perfdash"
+
+if [[ $# -lt 1 ]]; then
+  echo "Usage: $0 <benchmark-name> [tester args...]"
+  exit 1
+fi
+
+BENCHMARK_NAME="$1"
+ARGS="${*:2}"
+
+echo "Starting the etcd server..."
+./bin/etcd --data-dir="/tmp/" > /tmp/etcd.log 2>&1 &
```

Comment:
Create tmp subdirectory for data dir

Reply:
Sure, I've used the path `/tmp/etcd/data`.

Reply:
Please use `mktemp -d`